libs/feature/editor/src/lib/components/record-form/form-field/form-field-open-data/form-field-open-data.component.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnChanges,
+  Output,
+  SimpleChanges,
+} from '@angular/core'
+import { FormControl } from '@angular/forms'
+import { CheckToggleComponent } from '@geonetwork-ui/ui/inputs'
+import { getGlobalConfig } from '@geonetwork-ui/util/app-config'
+import { TranslateModule } from '@ngx-translate/core'
+
+@Component({
+  selector: 'gn-ui-form-field-open-data',
+  templateUrl: './form-field-open-data.component.html',
+  styleUrls: ['./form-field-open-data.component.css'],
+  standalone: true,
+  imports: [CheckToggleComponent, TranslateModule],
+})
+export class FormFieldOpenDataComponent implements OnChanges {
+  @Input() control: FormControl
+  value = false
+  @Output() visibilityChange = new EventEmitter<boolean>()
+
+  get config() {
+    return getGlobalConfig().LICENSES
+  }
+
+  ngOnChanges(changes: SimpleChanges) {
+    if ('control' in changes) {
```

Comment:
I'm working on this for the draft rollback, so not critical, but it's best to know, the formControl will never change, it's its value that's changing, and you detect that by subscribing on valueChanges.

Reply:
Good to know, will subscribe then :)